### PR TITLE
Fix typo in docs, code comments

### PR DESCRIPTION
**Pull request by sjfeng1999**
* fix typo in media/docs/layout.md
* fix typo in include/cutlass/arch/simd_61.h
* fix stride comment errors in TensorLayout

### CODE REVIEW

File: include/cutlass/layout/tensor.h [hunk 1]
```diff
@@ -312,7 +312,7 @@ class TensorNCxHWx {
   // Data members
   //
 
-  /// Stride data member - [c, wc, hwc]
+  /// Stride data member - [stride_w, stride_h, stride_n]
```
Comment (hwu36) [inline include/cutlass/layout/tensor.h:315]: This line needs to be `Interleave x w, Interleave x wh, hwc`

Comment (hwu36) [inline include/cutlass/layout/tensor.h:315]: This line needs to be `Interleave x w, Interleave x wh, hwc`

File: include/cutlass/layout/tensor.h [hunk 2]
```diff
@@ -421,7 +421,7 @@ class TensorCxRSKx {
...
```
Comment (hwu36) [inline include/cutlass/layout/tensor.h:424]: This line needs to be `Interleave x n, Interleave x nw, Interleave x nwh`

Comment (hwu36) [inline include/cutlass/layout/tensor.h:424]: This line needs to be `Interleave x n, Interleave x nw, Interleave x nwh`

**Comment by sjfeng1999** [comment]
I change `TensorNHWC` stride comment to `[c, wc, hwc]` together

**Review by hwu36 — APPROVED** [review]

**Pull request merged**